js/src/modals/CreateAccount/ChangeVault/changeVault.js

```diff
@@ -0,0 +1,51 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import { observer } from 'mobx-react';
+import React, { Component, PropTypes } from 'react';
+
+import { VaultSelect } from '~/ui';
+
+@observer
+export default class ChangeVault extends Component {
+  static propTypes = {
+    store: PropTypes.object.isRequired,
```

Comment:
No issues either way, but not going to do it just for the component and the others are out of scope for the PR. We should probably do it everywhere, we pass it everywhere as store. (Not just this component). Issues -

1. As you said, it may not be  obvious from reading what we are dealing with (fails a bit in "write code for reading")
2. It conflicts with Redux store naming, i.e. when a MobX store & Redux is used (couple of places), the MobX store cannot be called 'store' in props

Should be logged for a global refactor/cleanup - https://github.com/ethcore/parity/issues/4748